wpeverest/everest-forms
# Feature - Survey notice

### All Submissions:






### Changes proposed in this Pull Request:


A new notice i.e survey notice has been added which will be displayed if the user has activated plugin for more than 10 days and activated the EVF PRO license.

### How to test the changes in this Pull Request:

1. Verify if the survey notice design and contents are valid.
2. Verify if the survey notice is displayed as per the conditions.


### Types of changes:




### Other information:




### Changelog entry.

Feature - Survey notice
